configs/minimal/lightclient_patch.yaml

```diff
@@ -0,0 +1,8 @@
+# Minimal preset - lightclient patch
+
+CONFIG_NAME: "minimal"
+
+# ---------------------------------------------------------------
+
+SYNC_COMMITTEE_SIZE: 64
+SYNC_COMMITTEE_PUBKEY_AGGREGATES_SIZE: 16
```

Comment:
Are `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` and `DOMAIN_SYNC_COMMITTEE` suppose to be in config as well?